### PR TITLE
Feature/187876830 - Examples - Color filter

### PR DESCRIPTION
**Pull request by jeremymccurdy-redspace**
Ported over the color filter example.

Ticket: https://www.pivotaltracker.com/story/show/187876830

### CODE REVIEW

**Review by corinnejean — APPROVED** [review]

File: src/components/Resize/index.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  const [width, setWidth] = useState(0);
+  const [height, setHeight] = useState(0);
+  const [ratio, setRatio] = useState(0);
+
+  const [manager, setManager] = useState(null);
+  // SafeScaleManager uses the window object which isn't available in Node
+  // so the build process requires it to be dynamically imported to avoid errors
+  useEffect(() => {
+
+    if (typeof window !== 'undefined' && window.speechSynthesis) {
```
Comment (902seanryan) [inline src/components/Resize/index.tsx]: same note here about the speech synthesis

**Review by 902seanryan — CHANGES_REQUESTED** [review]

File: src/components/Resize/index.tsx (hunk 1 above)
Comment (jeremymccurdy-redspace) [reply src/components/Resize/index.tsx]: Ah, I think the branch is just out of date. Going to update it.

Comment (jeremymccurdy-redspace) [reply src/components/Resize/index.tsx]: @902seanryan I've updated the branch,

**Comment by 902seanryan** [comment]
@jeremymccurdy-redspace this is showing 0 changes now, were the changes included in the other PR?

**Comment by jeremymccurdy-redspace** [comment]
> @jeremymccurdy-redspace this is showing 0 changes now, were the changes included in the other PR?

Ugh, yes they were 🤦. Very sleep deprived today. That other PR was originally based off of this branch. I'm going to close out this one.

**Pull request closed without merging**